infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogEntryWriter.java

```diff
@@ -155,6 +146,8 @@ public long apply(LogReplicationEntryMsg msg) throws ReplicationWriterException
 
         // A new Delta sync is triggered, setup the new srcGlobalSnapshot and msgQ
         if (msg.getMetadata().getSnapshotTimestamp() > srcGlobalSnapshot) {
+            log.warn("setup the new srcGlobalSnapshot and lastMsgTs {}",
```

Comment:
Can this message be something a bit more significant or easy to follow in logs... Perhaps something like: "New log entry sync has started. Higher snapshot detected. prev={}, new={}" or something along those lines.

Reply:
done.